MIGRATING_TO_2.0.md

```diff
@@ -0,0 +1,4 @@
+# Migrating to V2
+
+- If you're using an older version of React, you'll have to upgrade to React 17.
+- You'll have to upgrade to Emotion 11. The main thing to notice is that you'll need to uninstall @emotion/core and install @emotion/react instead. More details [here ^](https://emotion.sh/docs/emotion-11){:target="\_blank"}
```

Comment:
Are you sure that this "target blank magic" works? It doesn't open in a new tab from this PR.

Reply:
Nah , will remove it.